core/src/autogluon/core/hpo/executors.py

```diff
@@ -7,14 +7,15 @@
 import pandas as pd
 import time
 
+import autogluon.common as ag
```

Comment:
I wonder if this is primarily a bad practice we are continuing from early days of repo. This `import ... as ag` logic was back when we would do `import autogluon as ag` before we were a namespace package.

For v1.0, the names and structure of `space.py` could maybe use a refactor.

Reply:
also, why aren't we doing `from autogluon.common import Space`?